client/config.xml

```diff
@@ -39,6 +39,10 @@
 
         <preference name="AndroidWindowSplashScreenAnimatedIcon" value="resources/splashscreen/android/splash_screen_icon_xxxhdpi.png" />
         <preference name="AndroidWindowSplashScreenBackground" value="#263237" />
+        <preference name="SplashScreenDelay" value="3000" />
```

Comment:
Please move the shared splash screen configuration outside the android block, and remove the old, unused settings, like the `<preference name="ShowSplashScreenSpinner" value="false" />` above and the `ShowSplashScreen` below.